test/rules-ceddl-fullstory.spec.ts

```diff
@@ -14,7 +14,7 @@ declare global {
   var digitalData: CEDDL;
 }
 
-describe('CEDDL to FullStory rules', () => {
+describe('Ruleset: CEDDL to FullStory', () => {
```

Comment:
Name change to enable a test grep pattern to include all converted ruleset tests (at this point, CEDDL and Google).